packages/xs/nbd.2.1.3/opam

```diff
@@ -27,13 +27,13 @@ depends: [
   "ounit" {test}
   "ocamlfind" {build}
   "ppx_tools" {build}
-  "lwt"
+  "lwt" {< "3.0.0"}
   "cstruct" {>= "1.9.0" & < "3.0.0"}
   "cmdliner"
   "sexplib"
   "mirage-block-unix"
   "io-page"
-  "mirage" {>= "1.1.0"}
+  "mirage" {>= "1.1.0" & < "3.0.0"}
```

Comment:
Please use https://github.com/ocaml/opam-repository/blob/master/packages/nbd/nbd.2.1.3/opam for nbd, I forgot to push the update to xs-opam